docs/addons/keda.md

```diff
@@ -0,0 +1,146 @@
+- `securityContextRunAsUser`: SecurityContext holds security configuration that will be applied to a container. runAsUser is the UID to run the entrypoint of the container process. This is an optional attribute, exposed directly inorder to make KedaScalar workaroud - https://github.com/kedacore/keda/issues/837#issuecomment-789037326
+- `securityContextRunAsGroup`: SecurityContext holds security configuration that will be applied to a container. rusAsGroup The GID to run the entrypoint of the container process. This is an optional attribute, exposed directly inorder to make KedaScalar workaroud - https://github.com/kedacore/keda/issues/837#issuecomment-789037326
+
+- `values`: Arbitrary values to pass to the chart. Refer to the Kubevious [Helm Chart documentation](https://github.com/kubevious/helm) for additional details
+
+## Validation
+
+To validate that Kubevious is installed properly in the cluster, check that the Kubevious deployments,
+services and stateful sets are running.
+
+```shell
+  kubectl get deployments -n keda  
+```
+There should be 2 deployments are created once for operator/agent and another for metrics server
+
+```shell
+  kubectl get pods -n keda  
+```
+There should be 2 pods one startin with name keda-operator- (Eg: "keda-operator-56bb8ddd5b-7fqb7")
+Verify IRSA is working properly
+```shell
+kubectl describe pods keda-operator-8c5967bcd-vlhpd  -n keda | grep -i aws  
+```
+
+```shell
+  kubectl get sa -n keda
+  kubectl describe sa keda-operator -n keda  
+```
+There should be an Service Account with name KedaOperator created and you can see an annotation on Service Account which have the AWS IAM Role ARN ( with required access for AWS Services like AWS SQS)
+
+## Testing
+
+We will test AWS SQS Scalar here
+1) Create SQS Queue in desired region
+```shell
+    aws sqs create-queue --queue-name keda-test --region REGION_NAME --output json
+```
+2) Create a deployment with SQS Consumer, For simplicity we will create an nginx deployment (in real case scenarion it should be an SQS Consumer)
+After the port-forwarding has started, the application can be accessed by navigating to http://localhost:8080
+```shell
+kubectl create ns sqs-consumer
+kubectl create deployment sqs-consumer --image nginx -n sqs-consumer
+```
+3) Create SQS Scalar by creating and applying following yaml file for eg: keda-scalar.yaml
+```yaml
+---
+apiVersion: keda.sh/v1alpha1 # https://keda.sh/docs/2.0/concepts/scaling-deployments/
+kind: ScaledObject
+metadata:
+  name: sqs-consumer-keda-scaler
+  namespace: sqs-consumer
+  labels:
+    app: sqs-consumer
+    deploymentName: sqs-consumer
+spec:
+  scaleTargetRef:
+    kind: Deployment
+    name: sqs-consumer
+  minReplicaCount: 1
+  maxReplicaCount: 50
+  pollingInterval: 10
+  cooldownPeriod:  500
+  triggers:
+  - type: aws-sqs-queue
+    metadata:
+      queueURL: https://sqs.AWS_REGION.amazonaws.com/ACCOUNT_NUMBER/sqs-consumer
```

Comment:
AWS_REGION is a replacement parameter here and on line 108. Users will run it as is unless we call it out. I suggest replace "AWS_REGION" with "${AWS_REGION}" and a statement before test to replace it. 
We had a couple of bugs reported on similar issues.

Reply:
Fixed